### ISSUE TITLE
REMissue

### ISSUE THREAD

**Issue by crwecker** [TwitchBronBron/brightscript-formatter#5]
Here is another one. If I have a variable name that starts with rem (removeSomething, remain etc.) then the following line fails to indent on an if statement:

```
    if (removeFoo <> invalid) then
    lineups["video"].push(invalid)
```

**Comment by TwitchBronBron** [comment]
Another good catch. I made the fixes and published npm version 1.1.3. Take a look and let me know how it goes!

**Comment by crwecker** [comment]
Worked great. Thanks!
